src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -1,14 +1,5 @@
 package com.easypost.model;
```

Comment:
Let's completely remove the `CreditCard` class and functionality. I know it will be a "version early", but it was added by accident and should be super low impact since it was a one-off. Let's just call it out in the CHANGELOG please.